lib/mixlib/install/backend/bintray.rb

```diff
@@ -152,14 +178,16 @@ def parse_platform_info(artifact_map)
           filename = artifact_map["name"]
           architecture = if %w{ x86_64 amd64 x64 }.fuzzy_include?(filename)
                            "x86_64"
-                         elsif %w{ i386 x86 }.fuzzy_include?(filename)
+                         elsif %w{ i386 x86 i86pc i686 }.fuzzy_include?(filename)
                            "i386"
-                         elsif %w{ powerpc }.fuzzy_include?(filename)
+                         elsif %w{ powerpc sun4u solaris2.5.10 solaris2.5.9 }.fuzzy_include?(filename)
                            "powerpc"
-                         elsif %w{ sparc }.fuzzy_include?(filename)
+                         elsif %w{ sparc sun4v }.fuzzy_include?(filename)
                            "sparc"
                          elsif platform == "mac_os_x"
                            "x86_64"
+                         elsif %w{ msi }.fuzzy_include?(filename)
+                           "i386"
                          else
```

Comment:
This is totally my ignorance, but do the `ppc64` and `ppc64le` architectures need to be represented in this list?